controllers/books.py

```diff
@@ -267,9 +267,6 @@ def _subchaptoc(course, chap):
         & (db.chapters.course_id == course)
         & (db.chapters.chapter_label == chap)
     ).select(
-        db.chapters.chapter_num,
```

Comment:
These are the lines that take away the numbers from the "This Chapter" menu.

If we can populate the chapter and subchapter table more accurately then we should do it.  Or if you have a different mechanism in mind to provide numbers here?